cineast-core/src/main/java/org/vitrivr/cineast/core/temporal/TemporalScoringAlgorithm.java

```diff
@@ -0,0 +1,84 @@
+package org.vitrivr.cineast.core.temporal;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+import org.vitrivr.cineast.core.data.StringDoublePair;
+import org.vitrivr.cineast.core.data.TemporalObject;
+import org.vitrivr.cineast.core.data.entities.MediaSegmentDescriptor;
+
+/**
+ * Abstract class with common objects and functions of all temporal scoring algorithms.
+ *
+ * <p>Assigns the necessary values and calculates preliminary maps</p>
+ *
+ * @author vGsteiger
+ * @created 12.05.2021
+ */
+public abstract class TemporalScoringAlgorithm {
```

Comment:
Please rename to `AbstractTemporalScoringAlgorithm`

Reply:
Resolved with the last commit